src/components/tab-group/tab-group.scss

```diff
@@ -0,0 +1,66 @@
+@import 'variables';
+@import 'default-theme';
+
+$tab-bar-height: 48px !default;
+
+:host {
+  display: block;
+  font-family: $md-font-family;
+  .md-tab-header {
+    border-bottom: 1px solid md-color($md-background, status-bar);
+  }
+}
+
+.md-tab-header {
```

Comment:
For each of the css classes here, could you add a brief overview of what they're for or which element/state they correspond to?